packages/js/e2e-environment/bin/post-results-to-github-pr.js

```diff
@@ -26,7 +34,8 @@ module.exports = async ( { github, context } ) => {
 
 		output = buildOutput( results );
 	} else {
-		output = `## Test Results Not Found!`;
+		output = `## Test Results Not Found! \n\n`;
+		output += 'The path to the `test-results.json` file may need to be updated in the `post-results-to-github-pr.js` script';
```

Comment:
I would end this at `updated.`. Once other teams start using the label they would not be expected to update the script.

Reply:
That's a fair point Ron. Message updated!